<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="4" hostname="LTIN251341" failures="0" tests="19" name="Test" time="2.625" errors="0" timestamp="2024-03-09T23:13:29 IST">
  <testcase classname="testBase.BaseClass" name="@BeforeTest setup" time="1.692">
    <failure type="org.openqa.selenium.SessionNotCreatedException" message="Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. 
Host info: host: &amp;apos;LTIN251341&amp;apos;, ip: &amp;apos;192.168.43.82&amp;apos;
Build info: version: &amp;apos;4.15.0&amp;apos;, revision: &amp;apos;1d14b5521b&amp;apos;
System info: os.name: &amp;apos;Windows 11&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;17.0.2&amp;apos;
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Command: [null, newSession {capabilities=[Capabilities {browserName: chrome, platformName: Windows 11}]}]
Capabilities {browserName: chrome, platformName: Windows 11}">
      <![CDATA[org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. 
Host info: host: 'LTIN251341', ip: '192.168.43.82'
Build info: version: '4.15.0', revision: '1d14b5521b'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.2'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Command: [null, newSession {capabilities=[Capabilities {browserName: chrome, platformName: Windows 11}]}]
Capabilities {browserName: chrome, platformName: Windows 11}
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:625)
at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:241)
at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:168)
at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:148)
at testBase.BaseClass.setup(BaseClass.java:88)
Caused by: java.io.UncheckedIOException: java.net.ConnectException
at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:450)
at org.openqa.selenium.remote.http.AddSeleniumUserAgent.lambda$apply$0(AddSeleniumUserAgent.java:42)
at org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:55)
at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute(JdkHttpClient.java:366)
at org.openqa.selenium.remote.tracing.TracedHttpClient.execute(TracedHttpClient.java:54)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:115)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:96)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:68)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:163)
at org.openqa.selenium.remote.TracedCommandExecutor.execute(TracedCommandExecutor.java:51)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:607)
... 32 more
Caused by: java.net.ConnectException
at java.net.http/jdk.internal.net.http.common.Utils.toConnectException(Utils.java:1047)
at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:198)
at java.net.http/jdk.internal.net.http.PlainHttpConnection.checkRetryConnect(PlainHttpConnection.java:230)
at java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$1(PlainHttpConnection.java:206)
at java.base/java.util.concurrent.CompletableFuture.uniHandle(CompletableFuture.java:934)
at java.base/java.util.concurrent.CompletableFuture$UniHandle.tryFire(CompletableFuture.java:911)
at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)
at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1773)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.nio.channels.ClosedChannelException
at java.base/sun.nio.ch.SocketChannelImpl.ensureOpen(SocketChannelImpl.java:195)
at java.base/sun.nio.ch.SocketChannelImpl.beginConnect(SocketChannelImpl.java:760)
at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:848)
at java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$0(PlainHttpConnection.java:183)
at java.base/java.security.AccessController.doPrivileged(AccessController.java:569)
at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:185)
... 9 more
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeTest setup -->
  <testcase classname="testBase.BaseClass" name="@AfterTest teardown" time="0.0">
    <skipped/>
  </testcase> <!-- @AfterTest teardown -->
  <testcase classname="testCases.TC_01_CabBooking" name="Handle_Popup" time="0.0">
    <skipped/>
  </testcase> <!-- Handle_Popup -->
  <testcase classname="testCases.TC_01_CabBooking" name="cabButton" time="0.001">
    <skipped/>
  </testcase> <!-- cabButton -->
  <testcase classname="testCases.TC_01_CabBooking" name="txt_From" time="0.001">
    <skipped/>
  </testcase> <!-- txt_From -->
  <testcase classname="testCases.TC_01_CabBooking" name="txt_To" time="0.001">
    <skipped/>
  </testcase> <!-- txt_To -->
  <testcase classname="testCases.TC_01_CabBooking" name="Date" time="0.001">
    <skipped/>
  </testcase> <!-- Date -->
  <testcase classname="testCases.TC_01_CabBooking" name="time" time="0.0">
    <skipped/>
  </testcase> <!-- time -->
  <testcase classname="testCases.TC_01_CabBooking" name="search" time="0.0">
    <skipped/>
  </testcase> <!-- search -->
  <testcase classname="testCases.TC_01_CabBooking" name="SUV" time="0.0">
    <skipped/>
  </testcase> <!-- SUV -->
  <testcase classname="testCases.TC_01_CabBooking" name="Printing_Lowest_price" time="0.0">
    <skipped/>
  </testcase> <!-- Printing_Lowest_price -->
  <testcase classname="testCases.TC_02_weddingCard" name="Gift_cards" time="0.0">
    <skipped/>
  </testcase> <!-- Gift_cards -->
  <testcase classname="testCases.TC_02_weddingCard" name="Wedding_Card" time="0.0">
    <skipped/>
  </testcase> <!-- Wedding_Card -->
  <testcase classname="testCases.TC_02_weddingCard" name="Emailbutton" time="0.0">
    <skipped/>
  </testcase> <!-- Emailbutton -->
  <testcase classname="testCases.TC_02_weddingCard" name="SendEmail" time="0.0">
    <skipped/>
  </testcase> <!-- SendEmail -->
  <testcase classname="testCases.TC_02_weddingCard" name="BuyButton" time="0.001">
    <skipped/>
  </testcase> <!-- BuyButton -->
  <testcase classname="testCases.TC_02_weddingCard" name="error" time="0.0">
    <skipped/>
  </testcase> <!-- error -->
  <testcase classname="testCases.TC_03_Hotels" name="hotels" time="0.0">
    <skipped/>
  </testcase> <!-- hotels -->
  <testcase classname="testCases.TC_03_Hotels" name="Guest" time="0.0">
    <skipped/>
  </testcase> <!-- Guest -->
  <testcase classname="testCases.TC_03_Hotels" name="adults" time="0.0">
    <skipped/>
  </testcase> <!-- adults -->
  <testcase classname="testCases.TC_03_Hotels" name="ExcelIntegration" time="0.0">
    <skipped/>
  </testcase> <!-- ExcelIntegration -->
  <testcase classname="testBase.BaseClass" name="@AfterTest teardown" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterTest teardown -->
  <testcase classname="testBase.BaseClass" name="@AfterTest teardown" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterTest teardown -->
  <testcase classname="testBase.BaseClass" name="@BeforeTest setup" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeTest setup -->
  <testcase classname="testBase.BaseClass" name="@BeforeTest setup" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeTest setup -->
</testsuite> <!-- Test -->
